src/test/java/io/supertokens/test/passwordless/PasswordlessConsumeCodeTest.java

```diff
@@ -701,8 +711,7 @@ public void testConsumeWrongLinkCodeExceedingMaxAttempts() throws Exception {
         for (int counter = 0; counter < maxCodeInputAttempts; counter++) {
 
             try {
-                Passwordless.consumeCode(process.getProcess(), createCodeResponse.deviceId,
-                        createCodeResponse.deviceIdHash, null, "n0p321");
+                Passwordless.consumeCode(process.getProcess(), null, null, null, "n0p321");
```

Comment:
The `deviceIdHash` should be defined both in the `deviceId`+`userInputCode` and the `linkCode` cases. It doesn't cause a problem here, because we throw but this should be the rule in general.

Reply:
resolved